include/cloudbreak.bash

```diff
@@ -158,6 +169,20 @@ gen-password() {
     date +%s | checksum sha1 | head -c 10
 }
 
+generate-cert() {
+    cloudbreak-config
+    if [ -f "${CBD_CERT_ROOT_PATH}/client.pem" ] && [ -f "${CBD_CERT_ROOT_PATH}/client-key.pem" ]; then
+      debug "Cloudbreak certificate and private key already exist, won't generate new ones."
+    else
+      info "generating Cloudbreak client certificate and private key in ${CBD_CERT_ROOT_PATH}"
+      docker run --rm -v ${CBD_CERT_ROOT_PATH}:/certs ehazlett/cert-tool:0.0.3 -d /certs -o=local &> /dev/null
+      cat "${CBD_CERT_ROOT_PATH}/ca.pem" >> "${CBD_CERT_ROOT_PATH}/client.pem"
```

Comment:
If the generated ca is not added when nginx is set up with the client cert, it won't be able to accept the client cert later because it won't be able to verify it up to a trusted root. See: http://trac.nginx.org/nginx/ticket/301